accounting/__init__.py

```diff
@@ -1,23 +1,15 @@
 # -*- coding:utf-8 -*
 #
-# Copyright 2016,2017
-# - Skia <skia@libskia.so>
```

Comment:
I'm not sure you're allowed to delete the original author of this file either. Adding the subsequent authors under the `AE UTBM` name should be OK, but you just can't erase the past like this.